internal/internal_task_pollers.go

```diff
@@ -159,6 +161,11 @@ type (
 		stopCh       <-chan struct{}
 		metricsScope *metrics.TaggedScope
 	}
+
+	// LocalDispatcher is an interface to dispatch locally dispatched activities.
+	LocalDispatcher interface {
```

Comment:
no reason for this to be public (AFAICT)

Reply:
Good point. Updated